test/TEST-13-NSPAWN-SMOKE/test.sh

```diff
@@ -70,6 +70,7 @@ if [[ -f /proc/1/ns/cgroup ]]; then
 fi
 
 is_user_ns_supported=no
+sysctl -w user.max_user_namespaces=10000
```

Comment:
could you add a comment here what this is about?

Reply:
Absolutely, should be fixed.